packages-web/badge/tests/e2e/specs/Badge.spec.ts

```diff
@@ -0,0 +1,65 @@
+import homepage from "../pages/home.page";
+
+describe("badge-web", () => {
+    beforeEach(() => {
+        homepage.open(); // resets page
+    });
+
+    describe("type: badge", () => {
+        it("should render a badge with a static caption", () => {
+            const caption = "Static";
+
+            const badge = homepage.badgeSuccess.getHTML();
+            expect(badge).toContain(caption);
+        });
+
+        it("renders with Bootstrap style success when style is success", () => {
+            const backgroundColor = homepage.badgeSuccess.getCSSProperty("background-color").value;
+            expect(backgroundColor).toContain("rgba(118,202,2,1)");
+        });
+
+        it("renders with Bootstrap style danger when style is danger", () => {
```

Comment:
Please add all the cases for the brand styles (inverse, warning, etc..)

Reply:
Done